docker-compose.yml

```diff
@@ -14,7 +14,8 @@ services:
       - ./data/db:/var/lib/mysql:cached
 
   saasform:
-    image: saasform/saasform:latest
+    image: saasform/saasform:localbuild
```

Comment:
Let's keep this file unchanged. We prefer people to pull from Docker Hub for now, as `main` may not be stable at points. All the images we build are based on releases and tested.

Reply:
I'm approving the PR and will do these update myself, to speed up